integreat_cms/core/settings.py

```diff
@@ -200,10 +200,10 @@
 )
 
 #: Path to the saved credential json file
-FCM_CREDENTIALS: Final[str | None] = os.environ.get("INTEGREAT_CMS_FCM_CREDENTIALS")
+FCM_CREDENTIALS: str | None = os.environ.get("INTEGREAT_CMS_FCM_CREDENTIALS")
 
 #: Whether push notifications via Firebase are enabled.
-#: This is ``True`` if :attr:`~integreat_cms.core.settings.FCM_KEY` is set, ``False`` otherwise.
+#: This is ``True`` if :attr:`~integreat_cms.core.settings.FCM_CREDENTIALS` is set, ``False`` otherwise.
 FCM_ENABLED: bool = bool(FCM_CREDENTIALS)
```

Comment:
Maybe it would make sense to only enable it here if the variable is a valid cred file which exists and is readable?
To me, it does not make sense that a dummy path of "dummy" which does not exist just enables the push news without any errors.
But ok, this also makes it harder to set dummy values :sweat_smile:

Reply:
Maybe we could add some warnings in case of invalid configuration?

But that should probably be done in another pr, because this would also apply to at least `GOOGLE_TRANSLATE_ENABLED`